pkg/api/types.go

```diff
@@ -5,6 +5,10 @@ import docker "github.com/fsouza/go-dockerclient"
 const (
 	DefaultNamespace    = "io.openshift.s2i."
 	KubernetesNamespace = "io.k8s."
+	DefaultIgnoreFile   = ".s2iignore"
+	DefaultUploadSubdir = "upload"
+	DefaultSrcSubdir = "src"
```

Comment:
Replace above two constants with `Source` from `api/scripts.go` and add there `upload/tmp` as well. And use those in the appropriate places in the code. I don't want to have multiple constants pointing to the same stuff.